README.md

```diff
@@ -1 +1,187 @@
+
+```java
+newApplicant.getFirstName();
+```
+
+## Resources
+
+All resources share the same interface when making API calls. Use `.create` to create a resource, `.find` to find one, and `.list` to fetch all resources.
+
+#### Applicants
+
+Applicants are the object upon which Onfido checks are performed.
+
+```java
+onfido.applicant.create(requestBody)                    # => Creates an applicant
+onfido.applicant.update("applicant_id", requestBody)    # => Updates an applicant
+onfido.applicant.destroy("applicant_id")                # => Schedule an applicant for deletion
+onfido.applicant.restore("applicant_id")                # => Restore an applicant scheduled for deletion
+onfido.applicant.find("applicant_id")                   # => Finds a single applicant
+onfido.applicant.list()                                 # => Returns all applicants
+```
+
+**Note:** Calling `onfido.applicant.destroy()` adds the applicant and all associated documents, photos, videos, checks, and reports to the deletion queue. They will be deleted 20 days after the request is made. An applicant that is scheduled for deletion can be restored but applicants that have been permanently deleted cannot.
+See https://documentation.onfido.com/#delete-applicant for more information.
+
+#### Documents
+
+Documents provide supporting evidence for Onfido checks.
+
+```java
+onfido.document.upload(inputStream, "fileName", requestBody) # => Creates a document
+onfido.document.find("document_id")                          # => Finds a document
+onfido.document.download("document_id")                      # => Downloads a document as a binary data
+onfido.document.list("applicant_id")                         # => Returns all applicant's documents
+```
+
+**Note:** The first parameter in the upload method should be an `InputStream` created from the desired file to upload.
+ 
+**Note:** The download method returns the file in the form of a `InputStream`.
+
+#### Live Photos
+
+Live Photos, like documents, can provide supporting evidence for Onfido checks.
+
+```java
+onfido.live_photo.upload(inputStream, "fileName", requestBody) # => Creates a live photo
+onfido.live_photo.find("live_photo_id")                        # => Finds a live photo
+onfido.live_photo.download("live_photo_id")                    # => Downloads a live photo as binary data
+onfido.live_photo.list("applicant_id")                         # => Returns all applicant's live photos
+```
+
+**Note:** The first parameter in the upload method should be an `InputStream` created from the desired file to upload.
+ 
+**Note:** The download method returns the file in the form of a `InputStream`.
+
+#### Checks
+
+Checks are requests for Onfido to check an applicant, by commissioning one or
```

Comment:
Very minor: "Commissioning" is exactly the right word but people with English as their second language might struggle. We could just use the text from documentation.onfido.com here instead, slightly altered:

> Checks are performed on an applicant. A check consists of one or more reports.